src/MongoDB.Driver/MongoUrlBuilder.cs

```diff
@@ -793,9 +793,19 @@ public override string ToString()
             {
                 query.AppendFormat("ipv6=true;");
             }
-            if (_useSsl)
+            if (_scheme == ConnectionStringScheme.MongoDBPlusSrv)
             {
-                query.AppendFormat("ssl=true;");
+                if (!_useSsl)
+                {
+                    query.AppendFormat("ssl=false;");
+                }
+            }
+            else
+            {
+                if (_useSsl)
+                {
+                    query.AppendFormat("ssl=true;");
+                }
```

Comment:
I discovered this additional required fix when adding more tests.

The idea is to only ouput ssl when the value is different from the default.l